src/util/serialize.cpp

```diff
@@ -354,6 +354,55 @@ std::string deSerializeJsonString(std::istream &is)
 	return os.str();
 }
 
+std::string serializeJsonStringIfNeeded(const std::string &s)
```

Comment:
Nevermind, I feel like this is a out of scope non-issue. If mods ever store enough meta to make this slow, they shouldn't be storing that much meta in an itemstack - given that it's stored in the player file